python/lsst/ts/watcher/rules/atcamera_dewar.py

```diff
@@ -0,0 +1,458 @@
+    median values reported within a configurable time window.
+    """
+
+    def __init__(self, config):
+        remote_info = watcher.RemoteInfo(
+            name="ATCamera",
+            index=0,
+            callback_names=["tel_vacuum"],
+            poll_names=[],
+        )
+        super().__init__(
+            config=config,
+            name="ATCameraDewar",
+            remote_info_list=[remote_info],
+        )
+        self.data_queue: salobj.BaseDdsDataType = collections.deque()
+        self.max_data_age: float = max(config.temperature_window, config.vacuum_window)
+        self.min_values = config.min_values
+        self.had_enough_data = False
+        self.threshold_handlers: typing.Dict[
+            str, typing.List[watcher.ThresholdHandler]
+        ] = collections.defaultdict(list)
+        # Measurement name: MeasurementInfo
+        self.name_meas_info = {
+            "ccd_temp": MeasurementInfo(
+                descr="CCD temperature",
+                field_name="tempCCD",
+                big_is_bad_list=[False, True],
+            ),
+            "cold_plate_temp": MeasurementInfo(
+                descr="Cold plate temperature",
+                field_name="tempColdPlate",
+            ),
+            "cryo_head_temp": MeasurementInfo(
+                descr="Cryo head temperature",
+                field_name="tempCryoHead",
+            ),
+            "vacuum": MeasurementInfo(
+                descr="Vacuum pressure",
+                field_name="vacuum",
+                is_temperature=False,
+            ),
+        }
+        # Dict of measurement name: alarm severity for that measurement.
+        # Used to provide the current_severity argument to
+        # ThresholdHandler.get_severity_reason.
+        self.name_severity = {name: AlarmSeverity.NONE for name in self.name_meas_info}
+
+        for name, info in self.name_meas_info.items():
+            for big_is_bad in info.big_is_bad_list:
+                self.threshold_handlers[name].append(
+                    self._make_threshold_handler(
+                        config=config, name=name, big_is_bad=big_is_bad
+                    )
+                )
+
+        # Support for start_call_counter
+        self.calls_remaining = 0
+        self.call_counter_task = utils.make_done_future()
+
+        # Task used to run and cancel no_data_timer.
+        self.no_data_timer_task = asyncio.Future()
+
+    def _make_threshold_handler(
+        self, config: types.SimpleNamespace, name: str, big_is_bad: bool
+    ) -> watcher.ThresholdHandler:
+        """Make a ThresholdHandler
+
+        Parameters
+        ----------
+        config : types.SimpleNamespace
+            Rule configuration.
+        name : str
+            Item is being measured. One of:
+            "ccd_temp", "cold_plate_temp", "cryo_head_temp", "vacuum".
+        big_is_bad : bool
+            Are larger values bad?
```

Comment:
I don't quite understand this. Can you maybe just add a sentence explaining this?

Reply:
I reworded it to:
```
big_is_bad : bool
    True if measured values larger than the specified levels are bad.
    False if measured values smaller than the specified levels are bad.
```
which more closely resembles the doc string for ThresholdHandler (which I also tweaked a bit, but it was already a lot clearer than what I wrote here).